packages/react-form-state/README.md

```diff
@@ -5,6 +5,8 @@
 
 Manage react forms tersely and safely-typed with no magic.
 
+Check out [@shopify/react-form](https://github.com/Shopify/quilt/tree/master/packages/react-form) for our newer form hooks.
```

Comment:
❤️ that you are adding this call out. If you can add some explanation to why someone would want to use `react-form` instead that would be even beter!

![tenor](https://user-images.githubusercontent.com/1610169/74861428-534b8480-5318-11ea-9773-d5d756025ace.gif)

Reply:
Attempt 2:

`This library is now superseded by [@shopify/react-form](https://github.com/Shopify/quilt/tree/master/packages/react-form) as it allows you to write the preferred, functional, and hooks-driven React components over class-based ones.`

Reply:
well, I mean to 👍 once the new text is in the code